climate_scanner/trends_innovation_classifier/src/fasttext/modules/trends.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+import os
+import yaml
+from .data_utils import load_params
+
+import glob
+import fasttext
```

Comment:
fasttext needs to be added to the requirements and setup.py file. Otherwise this causes an import error.